### ISSUE TITLE
WARNING:  Not compatible with latest airbnb ruleset

### ISSUE THREAD

**Issue by mikeerickson** [eslint/eslint#10534]
**Tell us about your environment**

* **ESLint Version:**

5.0.1

* **Node Version:**

10.4.1

* **npm Version:**

6.1.0

**What parser (default, Babel-ESLint, etc.) are you using?**

    "extends": [
        "eslint:recommended",
        "airbnb-base"
    ],

**Please show your full configuration:**

```
{
    "env": {
        "browser": true,
        "commonjs": true,
        "es6": true
    },
    "extends": [
        "eslint:recommended",
        "airbnb-base"
    ],
    "parserOptions": {
        "sourceType": "module"
    },
    "rules": {
        "indent": [
            "error",
            2
        ],
        "linebreak-style": [
            "error",
            "unix"
        ],
        "no-console": 0,
        "no-undef": 0
    }
}
```

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash

/node_modules/.bin/eslint 'resources/assets/js/**/*.js'



```

**What did you expect to happen?**

Did not expect to see warning

**What actually happened? Please include the actual, raw output from ESLint.**

```
(node:89840) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "airbnb-base")
```

**Comment by not-an-aardvark** [comment]
Thanks for the report. It's known that `eslint-config-airbnb` doesn't yet support ESLint v5 -- follow https://github.com/airbnb/javascript/issues/1834 for more details.

**Comment by mikeerickson** [comment]
Yes, I have since downgraded to ESLint 4.x, but figured it was worth noting her for other users who might not be aware of this issue
